packages/nc-gui/components/virtual-cell/Lookup.vue

```diff
@@ -49,7 +49,7 @@ const lookupColumnMetaProps = useColumn(lookupColumn)
 </script>
 
 <template>
-  <div class="h-full flex gap-1">
+  <div class="h-full flex gap-1 overflow-x-auto p-1">
```

Comment:
Wouldn't this allow horizontal scrolling? Is this desired?

Reply:
As I remember, if you got more lookup tags, they will be displayed out of the input window. So I made it overflow-x-auto atm.